### PR TITLE
Blueshield QOL with other code cleanup

### PR DESCRIPTION
**Pull request by Zergspower**
## About The Pull Request

What started as me just wanting to give the blueshield a weapon choice over the much disliked starter, led me into cleaning up the access list to make it far easier to read and to go into standards the rest of lists have.

The majority of the PR 'changes' aren't player facing and just updating to make things easier to read

What this changes:
**Gunset beacon for the Blueshield**
 - Bosgeo
 - Takbok
 - Hellfire Rifle that's 13 shots

And yes i tested the hellfire dont worry, in fact the 20 humans per gun below in this chart had a surprising addition! Husked corpses! Also damn the SC-1 is brutal

Against 20 naked humans
image

**Access Change:**
 - Added ORM access - why wasnt it there?
 - Added Cent general back - why was it removed?
 - Added EVA
 - Added Teleporter
 - Added All Lockers - same flag all command have
 - Replaced Research Access with Science Access - Research is for Ord and the likes, Science is for general science
 - Removes Announcement/Detective access

**NTC's Gunbeacon**
There's no difference in what was there before, i only added some descriptor since not everyone knows or cares to know the singular verb for a weapon. E.g. Takbok being a revolver

## How This Contributes To The Nova Sector Roleplay Experience

Gives people who severely hate the gun the blueshields got stuck with a few options, I wanted to add in the WT-550 as well but didnt want to test that one fully yet. Wont hurt to add later though.

Access - ORM access is there for everyone sans assistants, i think this was just overlooked. The removal of Cent_gen was also strange given they cant even enter the NTC's office before. Locker access mostly to return equipment of deceased head members, this doesnt add any access to get INTO the offices however.

## Proof of Testing
<details>
<summary>Screenshots/Videos</summary>

NTC door usable by Blueshield again
image

BS Beacon
image
image
 
NTC's beacon
image
 
</details>

## Changelog
:cl:
add: Added a gun beacon to the Blueshield Loadout, removing the locker spawn
add: Added a Hellfire Custom for the Blueshield to select as a weapon option
qol: Gun beacons have descriptions for the NTC's
qol: Blueshield now has acces to the ORM and NTC office
/:cl:

### CODE REVIEW

File: modular_nova/modules/blueshield/code/blueshield.dm [hunk 1]
```diff
@@ -69,3 +73,34 @@
...
+*/
+
+/// Blueshield's Custom Hellfire
+/obj/item/ammo_casing/energy/laser/hellfire/blueshield
+	projectile_type = /obj/projectile/beam/laser/hellfire
+	e_cost = LASER_SHOTS(13, STANDARD_CELL_CHARGE)
+	select_name = "maim"
+
+/obj/item/gun/energy/laser/hellgun/blueshield
+	name ="hellfire laser gun - Custom"
```
[the remark this answers is not in the document]
Comment (Zergspower) [reply modular_nova/modules/blueshield/code/blueshield.dm]: yeah that works better

**Comment by MajManatee** [comment]
Some things that would be nice but are arugably out of scope for this PR are:

The 550 that spawns in the locker being an option on the beacon
the praetorean MOD spawning in the suit holder whatever that most stations have (And currently holds a sec MOD for no reason)

Also what IS in scope is giving them basic cargo access. They have none.

**Comment by Zergspower** [comment]
> their cargo access was removed in the past because having that meant you could touch the order console

nothing was added for cargo, the list is just sorted to be easier to read.

**Comment by Ford-Defect** [comment]
As a frequent Blueshielder with my cowboy character, I very much appreciate the revolver choice.

**Comment by MaSvedish** [comment]
Would absolutely love more options like the WT, and maybe a shield? Pointman shield maybe as an option?

**Pull request merged**